### DISCUSSION TITLE
"Skipping canary release due to PR specifying no release"

### DISCUSSION THREAD

**Discussion by lloydjatkinson** [intuit/auto#2294] [Q&A]
![image](https://user-images.githubusercontent.com/5633938/212207938-95808a4b-c662-4447-bb74-6149d64aefa1.png)

The PR has a label. Yet, I only receive `Skipping canary release due to PR specifying no release. Use `auto canary --force` to override this setting.`

I don't believe `--force` should be required and it would be better if `shipit` worked as it's name implies. How do we make it run and take into account the labels?

**Comment by hipstersmoothie** [comment]
Can you most more of your logs?
